client/state/plugins/installed/reducer.js

```diff
@@ -33,6 +36,21 @@ export function isRequesting( state = {}, action ) {
 	}
 }
 
+/*
+ * Tracks the requesting state for installed plugins on a per-site index.
+ */
+export function isRequestingAll( state = {}, action ) {
```

Comment:
I see that the only reason to have this into Redux is this reducer. It also doesn't interact with other pieces of state. This makes the installed plugin requests a perfect candidate for `react-query`. That will save us a ton of Redux boilerplate.